setup.py

```diff
@@ -0,0 +1,36 @@
+import os
+from setuptools import find_packages, setup
+
+with open(os.path.join(os.path.dirname(__file__), 'README.md')) as readme:
+    README = readme.read()
+
+
+def get_install_requires():
+    """
+    parse requirements.txt, ignore links, exclude comments
+    """
+    requirements = []
+    for line in open('requirements.txt').readlines():
+        if line.startswith('#') or line == '' or line.startswith('http') or line.startswith('git'):
+            continue
+        requirements.append(line)
+    return requirements
+
+
+# allow setup.py to be run from any path
+os.chdir(os.path.normpath(os.path.join(os.path.abspath(__file__), os.pardir)))
+
+setup(
+    name='django-ipam',
+    version='0.1',   # Make it dynamic
```

Comment:
please import this like here: https://github.com/openwisp/django-freeradius/blob/master/setup.py#L39